test/instrumentation/modules/pg/_utils.js

```diff
@@ -1,12 +1,13 @@
 'use strict'
 
+var host = process.env.POSTGRES_HOST || 'localhost'
 var Client = require('pg').Client
 
 exports.reset = reset
 exports.loadData = loadData
 
 function reset (cb) {
-  var client = new Client({database: 'postgres'})
+  var client = new Client({host: host, database: 'postgres', user: 'postgres'})
```

Comment:
@watson I changed the user to `postgres` - which user is usually used when you run the tests locally?

Reply:
I use the default user which is either the value of the environment variable `PGUSER`, or if that isn't set, then the value of the environment variable `USER`. So in my case it's the username of the user I'm logged in as on my laptop.